tuf/builder.go

```diff
@@ -407,11 +427,15 @@ func (rb *repoBuilder) GenerateTimestamp(prev *data.SignedTimestamp) ([]byte, in
 	return sgndJSON, rb.repo.Timestamp.Signed.Version, nil
 }
 
+func (rb *repoBuilder) setPrevRoot(root *data.SignedRoot) {
```

Comment:
Group review:  This doesn't seem to be used, since in `LoadRootForUpdate` the `prevRoot` is just being set directly.  And additionally, both this function and the variable it's setting are unexported.